src/pms.cpp

```diff
@@ -744,7 +744,9 @@ int			Pms::init()
 	fieldtypes->add("trackshort", _("No"), FIELD_TRACKSHORT, 3, sort_compare_track);
 	fieldtypes->add("length", _("Length"), FIELD_TIME, 7, sort_compare_length);
 	fieldtypes->add("date", _("Date"), FIELD_DATE, 11, sort_compare_date);
+	fieldtypes->add("originaldate", _("Original date"), FIELD_ORIGINALDATE, 0, sort_compare_originaldate);
 	fieldtypes->add("year", _("Year"), FIELD_YEAR, 5, sort_compare_year);
+	fieldtypes->add("originalyear", _("Orig"), FIELD_ORIGINALYEAR, 5, sort_compare_originalyear);
```

Comment:
It's not a decision but a question. I'm trying to figure out what would be best for the majority of users. As suggested earlier, an `originaldate` setting would resolve this ambiguity.
I suggest leaving it as it is ("Orig") and then, if wanted, implementing this as an option in the future.